src/components/Landing/Hero.js

```diff
@@ -53,7 +53,7 @@ function Hero({ classes }) {
       <Grid item xs={6}>
         <Typography variant="h4" className={classes.intro}>
           We are here to give you actionable information about the quality of
-          your city&apos;s air, water and sound.
+          your city&apos;s air, water, sound and radiation level.
```

Comment:
@nerdlet Posts deployment, let us run this past editorial for copy editing.

Reply:
@chegejames  😞 We already have a feedback doc for this. You need to put this in there  and flag      it for editing

Reply:
@chegejames Edited the text.